doc/REQUIREMENTS.md

```diff
@@ -11,99 +11,102 @@ Note: "blind" is used in this document to mean “the contents of which are hidd
+- MAY be deleted in the "Empty" state
 
 ### Issuer
+
 > The entity that is requesting the creation and distribution capability of Packs from the PDS
 
-* MUST be represented by a Flow account
-* MUST have a sharable Cadence Capability for the creation of Pack NFT resources.
-* MUST be capable of transferring all required Collectible NFTs into the PDS
-* MAY or MAY NOT be the minter of the Collectibles
-* MAY or MAY NOT be the administrator of the PDS instance
+- MUST be represented by a Flow account
+- MUST have a sharable Cadence Capability for the creation of Pack NFT resources.
+- MUST be capable of transferring all required Collectible NFTs into the PDS
+- MAY or MAY NOT be the minter of the Collectibles
+- MAY or MAY NOT be the administrator of the PDS instance
 
 ### Owner
+
 > The entity that comes into possession of a Pack NFT
 
-* MUST be represented by a Flow account
-* MUST be able to receive the `PackReceiverCap` capability
-* MUST be able to `Reveal` the contents of an NFT pack
-* MUST be able to `Withdraw` Collectible NFTs from a successfully "Revealed" pack
+- MUST be represented by a Flow account
+- MUST be able to receive the `PackReceiverCap` capability
+- MUST be able to `Reveal` the contents of an NFT pack
+- MUST be able to `Withdraw` Collectible NFTs from a successfully "Revealed" pack
 
 ## Capabilities
 
 ### DistCap
+
 > Distribution capability
 
-* MUST be configured by the Issuer via the PDS before the capability is finally granted
-* MUST allow the issuer to transfer Pack NFTs.
-* MUST be lockable, i.e. during the Pack NFT creation period
-* MAY abort distribution at any time before a successful creation event
-* MUST provide an emergency abort function that can be triggered in certain circumstances
-e.g. the PDS malfunctions
-* SHOULD BE discarded after the creation period
+- MUST be configured by the Issuer via the PDS before the capability is finally granted
+- MUST allow the issuer to transfer Pack NFTs.
+- MUST be lockable, i.e. during the Pack NFT creation period
+- MAY abort distribution at any time before a successful creation event
+- MUST provide an emergency abort function that can be triggered in certain circumstances
+  e.g. the PDS malfunctions
+- SHOULD BE discarded after the creation period
 
 ### PackReceiverCap
+
 > NFT Pack Receiver Capability
 
-* MUST receive Pack NFTs after successful creation by the PDS
-* MUST expose the `Reveal` function on an owned Pack NFT, which:
-  * marks a Pack NFT as "Revealed"
-  * emits an on-chain event to be observed by the PDS with the Pack ID
-* MUST expose the `Withdraw` function on an owned Pack NFT, which:
-  * withdraws ALL Collectible NFTs from
+- MUST receive Pack NFTs after successful creation by the PDS
+- MUST expose the `Reveal` function on an owned Pack NFT, which:
+  - marks a Pack NFT as "Revealed"
+  - emits an on-chain event to be observed by the PDS with the Pack ID
+- MUST expose the `Withdraw` function on an owned Pack NFT, which:
```

Comment:
Just out of curiousity why `-` over `*`?

Reply:
Didn't intend to do that! I will revert those changes 👍

Reply:
Ah it is Prettier at it. Well we are going to face this someday so I'm rolling with it.